### PR TITLE
Create basic Benchmarks

### PR DESCRIPTION
**Pull request by mfigurski80**

### CODE REVIEW

File: benches/insert.rs [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+#[bench]
+fn bench_insert_platonic(b: &mut Bencher) {
+	b.iter(|| {
+		let mut map = RangeBoundsMap::new();
+		for i in 0..REPEAT {
+			let r = i..=i;
+			if let Err(OverlapError) = map.insert_platonic(r, i) {
+				panic!("Failed to insert");
+			}
```
Comment (ripytide) [inline benches/insert.rs:20]: This might be better looking with .expect("Failed to insert!")

File: benches/insert.rs [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+#[bench]
+fn bench_insert_coalesce_touching(b: &mut Bencher) {
+	b.iter(|| {
+		let mut map = RangeBoundsMap::new();
+		for i in 0..REPEAT / 2 {
+			let r1 = (10 * i)..(10 * i + 1);
+			let r2 = (10 * i + 1)..(10 * i + 2);
+			if let Err(e) = map.insert_coalesce_touching(r1, true) {
+				panic!("Failed to insert: {:?}", e)
+			}
```
Comment (ripytide) [inline benches/insert.rs:34]: Same here maybe.

**Review by ripytide — COMMENTED** [review]
Other than mild refactoring suggestions, this looks good :+1:

Thanks for the PR :sparkles:

**Comment by mfigurski80** [comment]
Hey @ripytide. Noted and fixed: thanks for catching that tbh. Coming back to the language after being somewhat green to begin with -- suggestions are def welcome.

**Pull request merged**